datajunction-query/djqs/engine.py

```diff
@@ -66,9 +66,30 @@ def get_columns_from_description(
     return columns
 
 
+def create_trino_engine(engine: Engine, headers: Optional[Dict[str, str]]) -> "Engine":
+    """
+    Create a SQLAlchemy engine for Trino.
+    """
+    uri = engine.uri
+    user = engine.extra_params["user"]
```

Comment:
Can these arguments to `extra_params` be pulled out to a set of constants at the top of the file? It'll be easier to refer to them and also see what the expected keys in `extra_params` are.